dss-core/src/main/java/io/github/ztkmkoo/dss/core/message/rest/DssRestResolverCommand.java

```diff
@@ -0,0 +1,71 @@
+package io.github.ztkmkoo.dss.core.message.rest;
+
+import io.github.ztkmkoo.dss.core.message.DssResolverCommand;
+import io.github.ztkmkoo.dss.core.network.DssChannelHandlerContext;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.util.Objects;
+
+/**
+ * @author Kebron ztkmkoo@gmail.com
+ * @create 2020-11-28 15:09
+ */
+public interface DssRestResolverCommand extends DssResolverCommand {
+
+    @Getter
+    class RestRequest extends DssRestCommand.RestRequest implements DssRestResolverCommand {
```

Comment:
May I ask why define classes in interface?

Reply:
Hi, (thank your review)

First of all, akka guide us define the command like follows
``` java
// from https://doc.akka.io/docs/akka/current/typed/actors.html#a-more-complex-example
static interface RoomCommand {}

public static final class GetSession implements RoomCommand {
  public final String screenName;
  public final ActorRef<SessionEvent> replyTo;

  public GetSession(String screenName, ActorRef<SessionEvent> replyTo) {
    this.screenName = screenName;
    this.replyTo = replyTo;
  }
}
```

And i think it is not a comman way,
so i declared the command interface and define the implement of the command in the class.

And when i define the class in the interface, I can access the `inner class` like DssRestResolverCommand.RestRequest, and it is much clear to me understand what kind of commands.
If I define in new .java file, I may need to make the class name like DssRestResolverRestRequestCommand. And it is so long.

Reply:
@Doyuni

Reply:
Thank you for answer :D 
I agree on this way, It's legible!